code/src/components/app-badge/AppBadge.tsx

```diff
@@ -0,0 +1,31 @@
+import Badge from "@mui/material/Badge";
+
+import { AppBadgeProps } from "@/components/app-badge/AppBadge.types";
+
+import cn from "@/utils/cn";
+
+import "@/components/app-badge/AppBadge.scss";
```

Comment:
not sure why do you need so many spaces between imported here, maybe can add spaces only for styles below and that's it like for our previous some code examples when we use space only for style import.
It's not urgent but still for future approaches nice to have